### DISCUSSION TITLE
How to synchronize data from tiptap servers and my data from the database that was cloned through a webhook?

### DISCUSSION THREAD

**Discussion by kobrynovych** [ueberdosis/tiptap#5275] [General Questions] [answered]
I use 'Collaboration extension', 'Comments extension' and 'Collaboration History extension'.

And I'm afraid that after some time the data on the tiptap servers will be deleted.
Please tell me how long the data will be stored on tiptap servers?

How can I synchronize the data ("documents", "comments" and "document versions") that are on the tiptap server with the data that we store on our server (we clone using a web hook)?

I will be very grateful for any hint, link or code fragment

**Comment by janthurau** [comment, marked as answer]
Hi @kobrynovych, you can fetch all data using our management API (https://tiptap.dev/docs/collaboration/documents/rest-api). You can also fetch all versions there (the API calls are documented in the postman collection).
